test/test_scanner.py

```diff
@@ -678,6 +689,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "libdb",
                         "11.2.5.3.21",  # Note that the full libdb version is longer than the package version
                     ),
+                    (
+                        "https://ftp.gnu.org/pub/gnu/ncurses/",
+                        "ncurses-6.1.tar.gz",
```

Comment:
Are there actually compiled ncurses binaries in this file?  Usually the gnu.org downloads are source only.  You may need to use a pre-compiled package from elsewhere to get this test to work.